images/coog/ep

```diff
@@ -14,141 +14,23 @@ COOG_IO=$COOG_ROOT/io
-    fi
-    if [ ! -z "$COOG_RQ_DB" ]
-    then
-        [ -z "$COOG_REDIS_HOST" ] && echo "no redis configured" && return 1
-        echo "rq = redis://$COOG_REDIS_HOST:$COOG_REDIS_PORT/$COOG_RQ_DB"
-    fi
-    return 0
-}
-
-hash_pwd() {
-    python -c "import crypt,random,string; print(crypt.crypt('$1', ''.join(random.sample(string.ascii_letters + string.digits, 8))))"
-}
-
-conf_session() {
-    echo "[session]"
-    [ -z "$COOG_SESSION_SUPERPWD" ] && return 0
-    local password
-    password=$(hash_pwd "$COOG_SESSION_SUPERPWD")
-    echo "super_pwd = $password"
-    return 0
-}
-
-conf_report() {
-    [ -z "$COOG_REPORT_API" ] && return 0
-    echo "[report]"
-    echo "api = $COOG_REPORT_API"
-}
-
-conf_sentry() {
-    echo "[sentry]"
-    [ ! -z "$COOG_SENTRY" ] && echo "dsn = $COOG_SENTRY"
-    return 0
-}
-
-conf_batch() {
-    echo "[batch]"
-    echo "root_dir = $COOG_IO/batch"
-}
-
-conf_test() {
-    [ -z "$COOG_TEST" ] && return 0
-    echo "[env]"
-    echo "testing = True"
-}
-
-_init_conf() {
-    # build conf from env vars
-    echo "" > "$COOG_CONF_SRV"
-    conf_postgres >> "$COOG_CONF_SRV" || return $?
-    conf_redis >> "$COOG_CONF_SRV" || return $?
-    conf_database >> "$COOG_CONF_SRV" || return $?
-    conf_cache >> "$COOG_CONF_SRV" || return $?
-    conf_async >> "$COOG_CONF_SRV" || return $?
-    conf_session >> "$COOG_CONF_SRV" || return $?
-    conf_report >> "$COOG_CONF_SRV" || return $?
-    conf_sentry >> "$COOG_CONF_SRV" || return $?
-    conf_batch >> "$COOG_CONF_SRV" || return $?
-    conf_test >> "$COOG_CONF_SRV" || return $?
-    # append override
-    local override
-    override="$COOG_IO/conf/coog.conf"
-    [ -f "$override" ] && cat "$override" >> $COOG_CONF_SRV
-    # set batch conf
-    touch "$COOG_CONF_BAT"
-    local batch_conf
-    batch_conf="$COOG_IO/conf/batch.conf"
-    [ -f "$batch_conf" ] && cat "$batch_conf" >> "$COOG_CONF_BAT"
-    return 0
-}
-
 _init_env() {
     export PYTHONUNBUFFERED=1
     export PYTHONPATH="$COOG_LIB:$COOG_TRYTOND:$PYTHONPATH"
     [ -d "$COOG_PROTEUS" ] && export PYTHONPATH="$COOG_PROTEUS:$PYTHONPATH"
     [ -z "$LOG_LEVEL" ] && LOG_LEVEL=ERROR
+    export DB_NAME="$COOG_DB_NAME"
     export LOG_LEVEL
-    export TRYTOND_CONFIG="$COOG_CONF_SRV"
-    export TRYTOND_BATCH_CONFIG="$COOG_CONF_BAT"
-    [ ! -z "$COOG_CELERY_DB" ] && export TRYTOND_ASYNC_CELERY="redis://$COOG_REDIS_HOST:$COOG_REDIS_PORT/$COOG_CELERY_DB"
-    [ ! -z "$COOG_RQ_DB" ] && export TRYTOND_ASYNC_RQ="redis://$COOG_REDIS_HOST:$COOG_REDIS_PORT/$COOG_RQ_DB"
-    if [ ! -z "$COOG_TEST" ]
-    then
-        DB_CACHE=/tmp
-        export DB_CACHE
-    elif [ ! -z "$COOG_DB_NAME" ]
-    then
-        export DB_NAME=$COOG_DB_NAME
-    fi
+    # Nowaday we need files to be physically created even if it is empty.
+    # (i.e: see coog-daily script)
+    # TODO: Remove totally depency to these files
+    echo "" > $COOG_CONF_SRV
```

Comment:
Un peu violent

Reply:
# TODO: Remove totally depency to these files